<ui version="4.0" >
 <class>DocumentProperties</class>
 <widget class="QDialog" name="DocumentProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Document Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>General</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineEditTitle" />
      </item>
      <item row="1" column="1" >
       <widget class="QTextEdit" name="lineEditAuthors" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>&amp;Title:</string>
        </property>
        <property name="buddy" >
         <cstring>lineEditTitle</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>&amp;Author(s):</string>
        </property>
        <property name="buddy" >
         <cstring>lineEditAuthors</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Native Language</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;Name:</string>
          </property>
          <property name="buddy" >
           <cstring>lineEditNativeLanguage</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lineEditNativeLanguage" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>&amp;Dictionary:</string>
          </property>
          <property name="buddy" >
           <cstring>comboBoxNativeDictionary</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBoxNativeDictionary" />
        </item>
        <item row="1" column="2" >
         <widget class="QToolButton" name="nativeDictionarySettings" >
          <property name="toolTip" >
           <string>Dictionary Settings</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Foreign Language</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>N&amp;ame:</string>
          </property>
          <property name="buddy" >
           <cstring>lineEditForeignLanguage</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lineEditForeignLanguage" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>D&amp;ictionary:</string>
          </property>
          <property name="buddy" >
           <cstring>comboBoxForeignDictionary</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="comboBoxForeignDictionary" />
        </item>
        <item row="1" column="2" >
         <widget class="QToolButton" name="foreignDictionarySettings" >
          <property name="toolTip" >
           <string>Dictionary Settings</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>lineEditAuthors</tabstop>
  <tabstop>lineEditNativeLanguage</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DocumentProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel" >
     <x>346</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
